File: backend/src/main/java/br/ufpr/sept/so2/modules/iam/api/dto/LoginRequest.java
package br.ufpr.sept.so2.modules.iam.api.dto;

import jakarta.validation.constraints.NotBlank;

public record LoginRequest(
        @NotBlank(message = "Informe e-mail ou GRR") String identificador,
        @NotBlank(message = "Informe a senha") String senha
) {
}

File: backend/src/main/kotlin/br/ufpr/sept/so2/modules/iam/api/dto/LoginRequest.kt
package br.ufpr.sept.so2.modules.iam.api.dto

import jakarta.validation.constraints.NotBlank

data class LoginRequest(
    @field:NotBlank(message = "Informe e-mail ou GRR")
    val identificador: String,
    @field:NotBlank(message = "Informe a senha")
    val senha: String,
)
